dmscripts/notify_suppliers_of_brief_withdrawal.py

```diff
@@ -0,0 +1,45 @@
+from dmutils.email.helpers import hash_string
+
+from dmscripts.helpers import env_helpers
+
+
+def get_brief_response_emails(data_api_client, brief_id):
+    responses = data_api_client.find_brief_responses(brief_id=brief_id, status="submitted").get("briefResponses")
+    return [response["respondToEmailAddress"] for response in responses]
+
+
+def create_context_for_brief(stage, brief):
+    return {
+        'brief_title': brief['title'],
+        'brief_link': '{0}/{1}/opportunities/{2}'.format(
+            env_helpers.get_web_url_from_stage(stage),
+            brief['frameworkFramework'],
+            brief['id']
+        )
+    }
+
+
+def main(data_api_client, mail_client, template_id, stage, logger, withdrawn_date=None, brief_id=None, dry_run=False):
+
+    withdrawn_briefs = data_api_client.find_briefs(withdrawn_on=withdrawn_date).get("briefs")
```

Comment:
Also, that endpoint paginates, so if there are more than 100 withdrawn briefs in a day (unlikely I know, but if we scaled massively it might be possible) it wouldn't send emails for every brief?

Reply:
Is this not an issue?